cpp/src/join/hash_join.cu

```diff
@@ -196,13 +196,18 @@ get_left_join_indices_complement(rmm::device_vector<size_type> &right_indices,
  * @throw std::out_of_range if elements of `build_on` exceed the number of columns in the `build`
  * table.
  *
- * @param build_table Table of build side columns to join.
+ * @param build_table Device view of table of build side columns to join.
+ * @param build Table of build side columns.
```

Comment:
What does "table of build side columns" mean? Do you mean "build-side columns"? What is a "build-side column"?  Please clarify this and what is different between this and the `build_table` parameter.

Reply:
I think I clarified this enough, but to be honest I'm somewhat lost in this nomenclature. Please let me know if you'd like more changes here.